app/Models/Event.php

```diff
@@ -113,7 +115,7 @@ public function achievements(): HasManyThrough
             'GameID',         // Achievements.GameID
             'achievement_id', // event_achievements.achievement_id
             'ID',             // Game.ID
-            'ID',             // Achievement.ID
+            'ID',             // Achievements.ID
```

Comment:
If you're going to fix this one, you should fix 117 as well: `GameData.ID`.

Reply:
I didn't even notice that one - fixed in latest.